File: CS/ImageMemoEdit/CustomGridControl/CustomGridInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.Text;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid;

namespace ImageTextEdit
{
    public class CustomGridInfoRegistrator : GridInfoRegistrator
    {
        public CustomGridInfoRegistrator() : base() { }
        public override BaseViewPainter CreatePainter(BaseView view) { return new CustomGridPainter(view as CustomGridView); }
        public override string ViewName { get { return "CustomGridView"; } }
        public override BaseView CreateView(GridControl grid){return new CustomGridView(grid);}
    }
}

File: VB/ImageMemoEdit/CustomGridControl/CustomGridInfoRegistrator.vb
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid

Namespace ImageTextEdit

    Public Class CustomGridInfoRegistrator
        Inherits GridInfoRegistrator

        Public Sub New()
            MyBase.New()
        End Sub

        Public Overrides Function CreatePainter(ByVal view As BaseView) As BaseViewPainter
            Return New CustomGridPainter(TryCast(view, CustomGridView))
        End Function

        Public Overrides ReadOnly Property ViewName As String
            Get
                Return "CustomGridView"
            End Get
        End Property

        Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
            Return New CustomGridView(grid)
        End Function
    End Class
End Namespace
